Cubical/HITs/Sn/Properties.agda

```diff
@@ -0,0 +1,105 @@
+{-# OPTIONS --cubical --safe #-}
+module Cubical.HITs.Sn.Properties where
+
+open import Cubical.Data.Int hiding (_+_)
+open import Cubical.Data.Bool
+open import Cubical.Foundations.Pointed
+open import Cubical.Foundations.Function
+open import Cubical.Foundations.Equiv
+open import Cubical.Foundations.HLevels
+open import Cubical.Foundations.Isomorphism
+open import Cubical.Foundations.Prelude
+open import Cubical.Foundations.GroupoidLaws
+open import Cubical.Foundations.Univalence
+open import Cubical.HITs.S1
+open import Cubical.Data.Nat
+open import Cubical.Data.Prod
+open import Cubical.HITs.Sn.Base
+open import Cubical.HITs.Susp
+open import Cubical.Data.Unit
+open import Cubical.HITs.Join
+open import Cubical.HITs.Pushout
+open import Cubical.HITs.SmashProduct
+
+private
+  variable
+    ℓ ℓ' : Level
+    A : Type ℓ
+    B : Type ℓ'
+
+
+
+--- Some silly lemmas on S1 ---
+
+S¹≡S1 : S₊ 1 ≡ S¹
+S¹≡S1 = cong Susp (sym (ua Bool≃Susp⊥)) ∙ sym S¹≡SuspBool
+
+isOfHLevelS1 : isOfHLevel 3 (S₊ 1)
+isOfHLevelS1 = transport (λ i → isOfHLevel 3 (S¹≡S1 (~ i)))
+                          λ x y → J (λ y p → (q : x ≡ y) → isProp (p ≡ q))
+                                     (transport (λ i → isSet (basedΩS¹≡Int x (~ i))) isSetInt refl)
+
+SuspBool→S1 : Susp Bool → S₊ 1
+SuspBool→S1 north = north
+SuspBool→S1 south = south
+SuspBool→S1 (merid false i) = merid south i
+SuspBool→S1 (merid true i) = merid north i
+
+S1→SuspBool : S₊ 1 → Susp Bool
+S1→SuspBool north = north
+S1→SuspBool south = south
+S1→SuspBool (merid north i) = merid true i
+S1→SuspBool (merid south i) = merid false i
+
+SuspBool≃S1 : Susp Bool ≃ S₊ 1
```

Comment:
Is it necessary to prove this so explicity? There is already https://github.com/agda/cubical/blob/master/Cubical/HITs/Susp/Base.agda#L62

Reply:
I did this to avoid having to use transports. I thought this way would compute better, but perhaps it makes no difference. Do you want me to change it or can I keep it?

Reply:
I don't think it really matters for this proof, but lets keep it as it is.